odd-platform-ui/src/components/Alerts/Alerts.tsx

```diff
@@ -41,6 +41,10 @@ const Alerts: React.FC<AlertsProps> = ({
 }) => {
   React.useEffect(() => {
     fetchAlertsTotals();
+    fetchAllAlertList({
+      page: 1,
+      size: 30,
+    });
```

Comment:
I think it would be better to place fetching of all alerts in the AlertsList component, where we pass the method to fetch. Also, due to the fact that we have pagination on the alerts page, it is better to tie through it.

Reply:
done